src/twisted/conch/ssh/transport.py

```diff
@@ -967,7 +967,7 @@ def ssh_DEBUG(self, packet):
         @type packet: L{bytes}
         @param packet: The message data.
         """
-        alwaysDisplay = bool(packet[0])
+        alwaysDisplay = bool(ord(packet[0:1]))
```

Comment:
This is correct on both Python 2 and 3 because both return `bytes` for _slices_ of `bytes`, and `ord` returns the ASCII integral value for single-item `bytes` objects on Python 3.